config/lint/.markdown-lint.yml

```diff
@@ -0,0 +1,27 @@
+---
+###########################
+###########################
+## Markdown Linter rules ##
+###########################
+###########################
+
+# Linter rules doc:
+# - https://github.com/DavidAnson/markdownlint
+#
+# Note:
+# To comment out a single error:
+#   <!-- markdownlint-disable -->
+#   any violations you want
+#   <!-- markdownlint-restore -->
+#
+
+###############
+# Rules by id #
+###############
+
+MD013: false
+MD025: false
+MD029: false
+MD033: false
+MD034: false
+MD036: false
```

Comment:
Why are we disabling these checks? The line length one is probably fine to disable, but we should perhaps fix the rest of the issues, unless there's a reason for not doing that (hard formatting, etc...).

Reply:
Done.